packages/core/src/exposed-thing.ts

```diff
@@ -131,7 +134,8 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
                 }
             }
             if (!languageSet) {
-                arrayContext.push({
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (arrayContext as any).push({
```

Comment:
Did a couple of tests and indeed my suspects were confirmed. TS correctly warns you about something nasty happening in this line. Basically, `arrayContext` at this point might be [] (for the typings point of view). Now to correctly satisfy the type you would first push a W3CUrlContext string and then your `@language` but TS gets confused cause it can't really tell if the array is empty or it has already the W3CUrlContext string. Sadly, checking for a not-empty array don't solve the issue (either TS is not smart enough or I'm missing something). So if we are already sure that there's already a W3CUrlContext inside thing["@context"] we can solve the error like this:
```ts
(arrayContext as Exclude<typeof arrayContext, []>).push({
                    "@language": TD.DEFAULT_CONTEXT_LANGUAGE,
                });
```

This would solve the TS error and removes the need of a cast to any. On the other hand, I wonder: should we check that the array is not empty and then push the correct string inside? Or at this point we already checked that? Also is this addition still relevant? do we need a default lang tag?